crates/chain/src/chain_data.rs

```diff
@@ -104,7 +104,7 @@ impl Default for BlockId {
     fn default() -> Self {
         Self {
             height: Default::default(),
-            hash: BlockHash::from_inner([0u8; 32]),
+            hash: BlockHash::from_slice(&[0u8; 32]).expect("len is 32"),
```

Comment:
Consider using `BlockHash::all_zeros()`.

Reply:
Also, `from_inner` was renamed to `from_byte_array`, so for other fixed arrays you still don't need to `.expect`.